spanner/spanner_snippets/spanner/spanner_directed_read_options.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/spanner"
+	sppb "cloud.google.com/go/spanner/apiv1/spannerpb"
+	"google.golang.org/api/iterator"
+)
+
+/*
+	Shows how to run a query with directed read options.
+	Only one of ExcludeReplicas or IncludeReplicas can be set
+	Each accepts a list of ReplicaSelections which contains Location and Type
+	* `location` - The location must be one of the regions within the
+	multi-region configuration of your database.
+	* `type` - The type of the replica
+	Some examples of using replica_selectors are:
+	* `location:us-east1` --> The "us-east1" replica(s) of any available type
+		will be used to process the request.
+	* `type:READ_ONLY`    --> The "READ_ONLY" type replica(s) in nearest
+	available location will be used to process the
+	request.
+	* `location:us-east1 type:READ_ONLY` --> The "READ_ONLY" type replica(s)
+	in location "us-east1" will be used to process the request.
+		IncludeReplicas also contains an option for AutoFailover which when set
+	Spanner will not route requests to a replica outside the
+	IncludeReplicas list when all the specified replicas are unavailable
+	or unhealthy. The default value is `false`
+*/
+// [START spanner_directed_read]
```

Comment:
issue: Move region tag to above `import` statement.

Reply:
Thanks. Done